installation/manual-installation/centos/README.md

```diff
@@ -19,12 +19,12 @@ nano /etc/yum.repos.d/mongodb-org.repo
 Paste this into the new file:
 
 ```
-[mongodb-org]
+[mongodb-org-4.0]
```

Comment:
We aren't actually ready to suggest 4.0 since its not officially supported by the framework we use

Reply:
Hi, apart from the mongo section which should stay with version 3.6, I notice you added some commands using sudo but the rest of the guide is without using sudo and that could be confusing, maybe remove the sudo? Thank you! Great change for the nodejs installation.

Reply:
The sudo was added as I was running into errors for those commands without it, even running as root.  I will spin up a clean VM this weekend and retest to verify.